py2neo/cypher/core.py

```diff
@@ -168,6 +173,41 @@ def execute_one(self, statement, parameters=None, **kwparameters):
             finally:
                 response.close()
 
+    def execute_for_visualisation(
```

Comment:
if this feature is implemented as a separate api (rather than looking in the `kwargs` of current apis) then i'll wrestle further for a better name.